packages/manager/modules/pci/src/projects/project/project.controller.js

```diff
@@ -49,6 +45,10 @@ export default class ProjectController {
       this.$timeout(() => angular.element('#sidebar-focus').focus());
     });
 
+    this.projectQuotaAboveThreshold = this.quotas.find(
+      (quota) => quota.quotAboveThreshold,
```

Comment:
```suggestion
      (quota) => quota.quotaAboveThreshold,
```
typo in API as well ?

Reply:
No only manager side. I will fix it.